src/components/proofs/ConnectAccount.tsx

```diff
@@ -40,14 +43,36 @@ const bottomWrapper = classnames(
 
 export default function () {
   const { walletLoading, needNetworkChange } = useSnapshot(WalletStore)
+  const zkProofText =
+    'In your wallet(s), you have NFTs that can point back to your identity (aka, getting doxxed). But with ZK proof, you can verify ownership of NFTs while staying pseudonymous.'
 
   return (
     <div className={walletContainer}>
       <div className={contentWrapper}>
         <HeaderText accent>First</HeaderText>
         <div className={bottomWrapper}>
           <BodyText center>
-            Connect a wallet with NFTs to create a ZK proof.
+            <span>
+              Connect a wallet with NFTs to create a{' '}
+              <AccentText color="text-accent">
+                <ToolTip position="bottom" fitContainer text={zkProofText}>
+                  <span
+                    className={classnames(
```

Comment:
We need to put those `classnames` into separate `const`

Reply:
ty, fixed